api/src/main/java/io/opencensus/trace/Span.java

```diff
@@ -144,8 +144,20 @@ public final void addAnnotation(String description) {
    * higher level applications.
    *
    * @param networkEvent the network event to add.
+   * @deprecated Use {@link #addMessageEvent} for higher level applications.
```

Comment:
Yes. I intend to explain why we deprecate the method (because another API for higher level application is provided). Maybe I should just omit it to avoid misunderstanding.

Reply:
I mean `addMessageEvent`, which is a more generalized API for higher level applications to use (including network application). The addNetworkEvent only applies to network application.

Anyway, I'll just use `Use {@link #addMessageEvent}.` to avoid misunderstanding.